.github/workflows/phpunit-l8.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 7.3]
+                php: [8.0, 7.4, 7.3]
```

Comment:
This file is about Laravel 8.

Seems like we left the 2 workflows (l7 and l8) by mistake. Can you remove the `phpunit-l8` workflow and rename the `l7` to `phpunit`. 

You can then also change the branch to v2 instead of master as we are in the v2 branch.

Reply:
Done. I had to test a bit about getting mockery running under php 8 without breaking any previous versions. Well, I think it's ok to install a minimum mockery "on demand", if the build target is php 8 as any local installation will do the same. Do you understand, why composer update is not installing the correct "lowest" if not explicitly mentioned? The package mockery/mockerey 1.4.1 seems to be the first with php 8 support.

Reply:
Got it. mockery/mockery < 1.4.0 had only php >= 5.6 as it's dependencies. Therefore it would be ok to install it as the lowest.